addons/trivy-operator/README.md

```diff
@@ -0,0 +1,74 @@
+# trivy-operator
+
```

Comment:
let's add some description for this addon, how to use？ Why we need it? What's benefit?